### PR TITLE
renamed client ID and secret -> application id and secret. group -> w…

### PR DESCRIPTION
**Pull request by may-hartov**
Renamed: (in AppOwnsData and UserOwnsData-> report sample)

* ClientID -> ApplicationID
* ClientSecret -> ApplicationSecret
* Group -> Worksapce

Improved Paas sample report and workspace id validation error message

### CODE REVIEW

File: User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0"?>
 <PBIWebApp.Properties.Settings>
-  <setting name="ClientID" serializeAs="String">
-    <value>{Enter your app ClientID}</value>
-  </setting>
-  <setting name="ClientSecret" serializeAs="String">
-    <value>{Enter your app SecretKey}</value>
+  <setting name="ApplicationID" serializeAs="String">
+    <value>d3bf2b56-81e4-4c3a-9b56-af9f226a6578</value>
```
Comment (ranbreuer) [inline User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config]: This should be empty!

File: User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   </setting>
   <setting name="PowerBiAPIResource" serializeAs="String">
     <value>https://analysis.windows.net/powerbi/api</value>
   </setting>
+  <setting name="ApplicationSecret" serializeAs="String">
+    <value>lcPzl8IrWAXEbHxd9uJMl3RDGbWP1p4Jo29rxuJ66IE=</value>
```
Comment (ranbreuer) [inline User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config]: Should be empty

File: User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config (hunk 1 above)
Comment (may-hartov) [reply User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config]: Removed

File: User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config (hunk 2 above)
Comment (may-hartov) [reply User Owns Data/integrate-report-web-app/PBIWebApp/Cloud.config]: Removed

**Pull request merged**